api/src/db/db.service.ts

```diff
@@ -102,28 +115,63 @@ export class DbService {
     }
 
     /**
-     * Get data to which a user has access and has subscribed to
-     * @param {Array<string>} accessTags - Array with access tag ID's
-     * @param {Array<string>} subscriptionTags - Array with subscription tag ID's
+     * Function used to clear a database
+     */
+    async destroyAllDocs() {
+        const res = await this.db.list();
+        const pList = [];
+        res.rows.forEach((doc) => {
+            pList.push(this.db.destroy(doc.id, doc.value.rev));
+        });
+
+        await Promise.all(pList);
+    }
+
+    /**
+     * Get data to which a user has access to including the user document itself.
+     * @param {string} userID - User document ID.
+     * @param {getDocsOptions} options - Query configuration object.
      * @returns - Promise containing the query result
      */
-    getDocs(accessTags: Array<string>, subscriptionTags: Array<string>) {
+    getDocs(userId: string, options: getDocsOptions): Promise<unknown> {
+        // Set default options
+        if (!options.from) options.from = 0;
```

Comment:
Currently this is mandatory in the type, would probably make sense to make it optional (and add a test for this).